x-pack/test/alerting_api_integration/security_and_spaces/group1/tests/alerting/backfill/delete.ts

```diff
@@ -23,9 +24,18 @@ export default function deleteBackfillTests({ getService }: FtrProviderContext)
   const supertestWithoutAuth = getService('supertestWithoutAuth');
 
   describe('delete backfill', () => {
+    let backfillIds: Array<{ id: string; spaceId: string }> = [];
     const objectRemover = new ObjectRemover(supertest);
 
-    after(() => objectRemover.removeAll());
+    afterEach(async () => {
```

Comment:
This cleans up backfills that were scheduled but not deleted in the "user has no permissions to delete" tests.